docs/fsharp/using-fsharp-on-azure/file-storage.md

```diff
@@ -36,59 +42,51 @@ For the tutorial, you'll enter your connection string in your script, like this:
 
 [!code-fsharp[FileStorage](../../../samples/snippets/fsharp/azure/file-storage.fsx#L14-L14)]
 
-However, this is **not recommended** for real projects. Your storage account key is similar to the root password for your storage account. Always be careful to protect your storage account key. Avoid distributing it to other users, hard-coding it, or saving it in a plain-text file that is accessible to others. You can regenerate your key using the Azure portal if you believe it may have been compromised.
-
-For real applications, the best way to maintain your storage connection string is in a configuration file. To fetch the connection string from a configuration file, you can do this:
-
-[!code-fsharp[FileStorage](../../../samples/snippets/fsharp/azure/file-storage.fsx#L16-L18)]
-
-Using Azure Configuration Manager is optional. You can also use an API such as the .NET Framework's `ConfigurationManager` type.
-
 ### Create the File service client
```

Comment:
This is some odd capitalization.   Why is `File` capitalized?

Reply:
It’s the original format, we have overlooked this detail. You are right, lowercase should be used here.